doc/classes/EditorSettings.xml

```diff
@@ -319,6 +325,10 @@
 		<member name="editors/3d/grid_yz_plane" type="bool" setter="" getter="">
 			If [code]true[/code], render the grid on a YZ plane. This can be useful for 3D side-scrolling games.
 		</member>
+		<member name="editors/3d/manipulator_gizmo_opacity" type="float" setter="" getter="">
+		</member>
+		<member name="editors/3d/manipulator_gizmo_size" type="int" setter="" getter="">
+		</member>
```

Comment:
```suggestion
		<member name="editors/3d/manipulator_gizmo_size" type="int" setter="" getter="">
			Size of the default gizmo for moving, rotating, and scaling 3D nodes.
		</member>
```
No idea what units it uses. The value *roughly* corresponds to the gizmo size in pixels when using orthogonal mode.